### PR TITLE
timing structures added to the UI

### LINKED ISSUE #5773: Add timing chart to somewhere in the Jinteki.net UI
Maybe in a tab in the log? maybe as a full main tab on the top? Maybe as a background image? I dunno! 

but I'm adding this just because I've talked to someone before about this, but was just reminded that all of those discussions are ephemeral, it's the Github Issue list that is forever

### LINKED ISSUE #6555: improvement? timing structure of a run+breach diagram
Hello,

I think that having the whole timing structure of a run + breach diagram available in-game on jnet would be a useful thing. Would it be possible to add an option next to Game Log and Settings (over the chat window) so that the timing structure of a run and a breach could be printed out in all of their glory in a dedicated panel (like the in-game chat and settings option do)?

Extra points if it highlighted the current stage during a run.

As always, thank you very very much for you work!

### PR DESCRIPTION
**Pull request by NBKelly**
Should hopefully make it easier for some players to understand the game.

For clarity/readability, this isn't exactly linguistically 1:1 with the CR.

There's probably some capitalization/punctuation/apostrophes to clean up too. We can just hit those as they get pointed out.

Good luck to anyone who wants to translate this. That's not my problem :joy:.

Run Timing:
timing_000
timing_001

Turn Timing:
turn-timing
turn-timing_000

Closes #5773
Closes #6555

### CODE REVIEW

File: src/cljs/nr/gameboard/diagrams.cljs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+   [nr.appstate :refer [app-state]]
+   [nr.translations :refer [tr]]
+   [nr.utils :refer [render-icons]]))
+
+(defn- bullet
+  [tag text]
+  (let [tag (nth "abcdefghi" tag)]
+    [:div
+     [:td [:label (str tag ") ")]]
+     [:td [:div text]]]))
```
Comment (NoahTheDuke) [inline src/cljs/nr/gameboard/diagrams.cljs:12]: any reason to not use `<li>`?

**Comment by NoahTheDuke** [comment]
Hell yeah, great idea. Could move the strings out to the translation file, so other languages can adapt as desired.

File: src/cljs/nr/gameboard/diagrams.cljs (hunk 1 above)
Comment (NBKelly) [reply src/cljs/nr/gameboard/diagrams.cljs:12]: Honestly, I simply forgot :joy:.

**Comment by NBKelly** [comment]
I added in all the translation stuff, and filled out the entries in the english section.

I don't have the skillset to figure out the html/css stuff to make it look nice as a list though, so I think that's a problem for somebody else later if it bothers them :joy:

File: src/cljs/nr/gameboard/diagrams.cljs (hunk 1 above)
Comment (NoahTheDuke) [reply src/cljs/nr/gameboard/diagrams.cljs:12]: cool, no worries

**Pull request merged**